source/tutorials/multi-boot/multi-boot.rst

```diff
@@ -0,0 +1,99 @@
+.. _multi-boot:
+
+Multi-boot |CL-ATTR| and Windows\* OS
+#####################################
+
+This guide shows how to install |CL-ATTR| adjacent to an existing Windows\*
+OS installation. To add |CL| to an existing Windows installation, there are two possible methods:
+
+.. contents::
+   :local:
+   :depth: 1
+
+Method 1: Shrink the Windows partition to make space for |CL|
+*************************************************************
+
+#. Boot up Windows OS 10.
+
+#. Launch the :file:`Disk Management` utility found under
+   :guilabel:`Create and format hard disk partitions`.
+
+#. Right-click the primary Windows partition and select “Shrink Volume...”.
+
+   .. figure:: ../../_figures/multi-boot/multi-boot-win-01.png
+      :width: 75%
+      :alt: Disk Management > Shrink Volume
+
+      Figure 1: Disk Management > Shrink Volume.
+
+#. Shrink the size of the partition by at least the following amount:
+
+   * For :ref:`bare-metal-install-desktop`, allow at least 20GB.
+
+   * For :ref:`bare-metal-install-server`, allow at least 4GB.
+
+#. In this tutorial, We follow :ref:`bare-metal-install-desktop`.
+   We shrink C from about 91000 to 71000, as shown in Figure 2.
+
+   .. figure:: ../../_figures/multi-boot/multi-boot-win-02.png
+      :width: 75%
+      :alt: Shrink C
+
+      Figure 2: Shrink C.
+
+#. Shutdown Windows OS.
+
+#. Follow the instructions to :ref:`bare-metal-install-desktop`.
+
+#. After booting from the |CL| image, select the icon to
+   launch the installer.
+
+#. Click :guilabel:`Select Installation Media`.
+
+#. Select :guilabel:`Safe Installation`.
+
+#. Go through the remaining steps to install |CL|.
+
+#. Shut down your system and remove the USB.
+
+#. Reboot.
+
+#. During the BIOS POST stage, press the proper F-key for your system to
+   bring up the “Boot Menu”.
+
+#. Now, select the Windows partition (“Windows Boot Manager”) to change
+   boot to Windows OS, or select the other one to boot |CL|.
+
+   .. note::
+
+   	  Some BIOSes do not support multiple partitions.
+
+#. Select :kbd:`F10` or your system default to save the primary
```

Comment:
lines 71-72: I'm assuming you mean press F10 to bring up the boot menu and not "save"?  Regardless,  I do not think this line is needed the previous line already tells the user to press the appropriate F-key to bring up the boot menu.